### DISCUSSION TITLE
Pimcore v10.5.17 - Why Add thumbnails button are disabled for admin user

### DISCUSSION THREAD

**Discussion by bumim10** [pimcore/pimcore#15508] [General]
Hi team, 

we have found that some buttons [i.e. Add in Thumbnails settings, Add in Custom reports] for admin user are disabled.
![image](https://github.com/pimcore/pimcore/assets/139216913/33661ae4-1ff8-4c94-a995-c8cd68ecf9eb)
Please let me know why I can't use this function. 
Any help would be appreciated

**Comment by kingjia90** [comment]
Hi,

if you have configured `PIMCORE_WRITE_TARGET_IMAGE_THUMBNAILS` [store location](https://pimcore.com/docs/pimcore/10.3/Development_Documentation/Deployment/Configuration_Environments.html#page_Configuration-Storage-Locations-Fallbacks) to be not writable or the type selected doesn't allow to be edited (eg. yaml) or the physical file unix permission is not properly set, then it would appear like this
